### PR TITLE
fix: added optional inputs

### LINKED ISSUE #1164: MailChimp first and last names (optional inputs)
I'm writing an article about lead generation with Google Forms, MailChimp and HubSpot. When the user enters their info in the form, I can only add their email to MailChimp today.

I'd like to add the first name and last name too. Maybe more optional inputs as well.

### PR DESCRIPTION
**Pull request by abdullahranginwala**
## What does this PR do?

Adds first name and last name as optional inputs in the mailchimp piece.

Fixes #1164

### CODE REVIEW

File: packages/pieces/mailchimp/src/lib/actions/add-member-to-list.action/add-member-to-list.action.ts [hunk 1]
```diff
@@ -9,6 +9,16 @@ export const addMemberToList = createAction({
     description: "Add a member to an existing Mailchimp audience (list)",
     props: {
         authentication: mailChimpAuth,
+        first_name: Property.ShortText({
+            displayName: 'First Name',
+            description: 'First name of the new contact',
+            required: false,
+        }),
+        last_name: Property.ShortText({
```
Comment (abuaboud) [inline packages/pieces/mailchimp/src/lib/actions/add-member-to-list.action/add-member-to-list.action.ts:17]: Can you please send the data to mailchimp if they existed?

Comment (abdullahranginwala) [reply packages/pieces/mailchimp/src/lib/actions/add-member-to-list.action/add-member-to-list.action.ts:17]: Sure! I'll make the changes

Comment (abdullahranginwala) [reply packages/pieces/mailchimp/src/lib/actions/add-member-to-list.action/add-member-to-list.action.ts:17]: I have to send the first name and last name along with the email if those particular optional inputs were added. Correct?

Comment (abuaboud) [reply packages/pieces/mailchimp/src/lib/actions/add-member-to-list.action/add-member-to-list.action.ts:17]: Yes

**Pull request closed without merging**